src/main/java/org/opensearch/index/codec/customcodecs/Lucene95CustomStoredFieldsFormat.java

```diff
@@ -132,4 +132,14 @@ Lucene95CustomCodec.Mode getMode() {
     public int getCompressionLevel() {
         return compressionLevel;
     }
+
+    public Map<Lucene95CustomCodec.Mode, CompressionMode> getCompressionModes(){
+        return this.compressionModeMap;
+    }
+
+    private void populateCompressionModeMap(int compressionLevel) {
```

Comment:
This will help whenever we will plan to add more compression modes, and then we will not have to create separate data members.